index.html

```diff
@@ -0,0 +1,124 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+
+<div id="page-wrapper">
+  <header id="header">
+    <div class="logo">
+      <img
+        id="header-img"
+        src="images/metamask.png"
+      />
+    </div>
+
+    <nav id="nav-bar">
+      <ul>
+        <li><a class="nav-link" href="#features">Product</a></li>
+        <li><a class="nav-link" href="#how-it-works">How To Buy</a></li>
+        <li><a class="nav-link" href="#pricing">Price</a></li>
+      </ul>
+    </nav>
+  </header>
+
+  <div class="container"></div>
+
+  <section id="hero">
+    <h2>Find a Dog you Wish</h2>
+    <form id="form" action="https://www.freecodecamp.com/email-submit">
+      <input
+        name="email"
+        id="email"
+        type="email"
+        placeholder="Enter your email address"
+        required
+      />
+      <input id="submit" type="submit" value="Get Started" class="btn" />
+    </form>
+  </section>
+
+  <div class="container">
+    <section id="features">
+      <div class="grid">
+        
+        <div class="desc">
+          <h2>afador</h2>
+          <p>
+           Lorem, ipsum dolor sit amet consectetur adipisicing elit. Accusamus, eum.
+          </p>
+        </div>
+      </div>
+      <div class="grid">
+        
+        <div class="desc">
+          <h2>Akita pit</h2>
+          <p>
+           Lorem ipsum, dolor sit amet consectetur adipisicing elit. Aspernatur, laudantium?
+          </p>
+        </div>
+      </div>
+      <div class="grid">
+        
+        <div class="desc">
+          <h2>Pitbull</h2>
+          <p>
+            Lorem ipsum dolor sit amet consectetur adipisicing elit. Mollitia, quas.
+          </p>
+        </div>
+      </div>
```

Comment:
for semantic purposes please don't use too many div rather try to use sections, headers, kindly go ahead and change the main div here to section tags or main tags and change that to an article tag.

Reply:
Great job 👍 identifying the problem but you could write this in a more polite manner as `" Kindly make sure to use section tags, headers tags, for semantic purposes please avoid using too many div tags. Make sure to change the main div here to section tags or main tags ."` Please make sure you don't use the word `don't` as it hypers the student. Make sure to use words such as `kindly`, `make sure` with a proper sentence to make the student work tension free.